server/db/models/products.js

```diff
@@ -0,0 +1,44 @@
+//define Products model here
+const Sequelize = require('sequelize');
+const { db } = require('./../database.js');
```

Comment:
the bracket notation is not needed here, I believe:

**const db  = require('./../database.js');**